packages/cli/src/zosjobs/-strings-/en.ts

```diff
@@ -73,6 +73,43 @@ export default {
             }
         }
     },
+    SEARCH: {
+        SUMMARY: "Search the spool output of a z/OS job",
+        DESCRIPTION: "Search the spool output of a job.",
+        ACTIONS: {
+            JOB: {
+                SUMMARY: "Search the spool output of a job.",
+                DESCRIPTION: "Search the spool output of a job.",
+                POSITIONALS: {
+                    JOBNAME: "The job name to search. Wildcards are accepted for the job name." +
+                        " You can use, for example \"USERJOB*\" to search all jobs that start" +
+                        " with USERJOB."
+                },
+                OPTIONS: {
+                    SEARCHSTRING: "The string to search for in the spool output",
+                    SEARCHREGEX: "The regular expression to search for in the spool output",
+                    CASEINSENSITIVE: "The search is case insensitive or not",
+                    SEARCHLIMIT: "The maximum number of matching lines to return for an individual spool file",
```

Comment:
Hi @KevinLoesch1, could you add a period at the end of these option definitions?

Reply:
I have checked in an update to correct this.